README.md

```diff
@@ -231,12 +222,16 @@ awesome_startup: dev, production, staging
 
 ### Execute arbitrary Capistrano tasks
 
-You can launch any Capistrano task via `task` command. You can also
-pass arguments in _Rake style_ (i.e. via square brackets after task name)
+You can launch any Capistrano task via `task` command.
 
 ```
-$ pulsar task mycustom:task[argumen1,argument2] my_application staging
+$ pulsar task my_application staging mycustom:task
+```
+You can also pass arguments by wrapping the task and arguments in quotes.
+
 ```
+$ pulsar task my_application staging "mycustom:task TASK_ARG1=arg1 TASK_ARG2=arg2"
```

Comment:
Hm I suppose they are, I was just updating that section since the square bracket way of passing arguments errors out and says "no matches found". I'll update it for the example to be --dry-run as the argument in quotes like in #58 ?

Reply:
`pulsar task my_application staging "mycustom:task[argument1,argument2]"` this ordering works, with the task at the end. Updating again, sorry!

Reply:
@mtylty Updated that section to read like it originally did just with the ordering updated. Trying the order currently in there throws `Failed to execute task staging for mycustom:task[argumen1,argument2] on my_application`

Reply:
@octoxan I don't get it 😅 are you saying it's working or it is not?

Reply:
@mtylty I mean it wasn't working with the commands in the current readme, but it is working with the commands in this updated one lol. Sorry! Now all the commands in the readme will work. Just had to reorder the command so the task name is last after the stage.